test/translation/ts/dotColonFunctionCalls.ts

```diff
@@ -0,0 +1,24 @@
+declare class TestClass {
+    public dotMethod: () => void;
+    public colonMethod(): void;
+}
+
+declare interface TestInterface {
+    dotMethod: () => void;
+    colonMethod(): void;
+}
+
+declare namespace TestNameSpace {
+    var dotMethod: () => void;
+    function dotMethod2(): void;
+}
+
+declare const classInstance: TestClass;
+declare const interfaceInstance: TestInterface;
+
+classInstance.colonMethod();
+classInstance.dotMethod();
+interfaceInstance.colonMethod();
+interfaceInstance.dotMethod();
+TestNameSpace.dotMethod();
+TestNameSpace.dotMethod2();
```

Comment:
Not sure how much sense it makes but maybe add some functional tests too. Probably not that important.